sdk/messaging/azservicebus/internal/stress/Chart.yaml

```
@@ -0,0 +1,14 @@
+apiVersion: v2
+name: go-sb-stress
+description: Stress tests for Go
+version: 0.1.1
+appVersion: v0.1
+annotations:
+  stressTest: 'true'  # enable auto-discovery of this test via `find-all-stress-packages.ps1`
+  namespace: 'go-sb-stress'
```

I had been thinking that namespaces would be better to mirror the language repo for organization purposes?

```suggestion
  namespace: 'golang'
```

---

Done, but just used 'go'.